megalist_dataflow/uploaders/google_ads/conversions/google_ads_offline_conversions_uploader.py

```diff
@@ -30,13 +30,13 @@ def __init__(self, oauth_credentials, developer_token):
     self.active = self.developer_token is not None
 
   def _get_ads_service(self, customer_id: str):
-    return utils.get_ads_service('GoogleAdsService', 'v7',
+    return utils.get_ads_service('GoogleAdsService', 'v8',
```

Comment:
isn't it better is we have a single constant somewhere for this value?

Reply:
Sounds good